variables.tf

```
@@ -200,3 +200,9 @@ variable "transport_protocol" {
     error_message = "Invalid protocol type must be one of: udp, tcp."
   }
 }
+
+variable "permissions_boundary" {
```

Related to https://github.com/cloudposse/terraform-aws-ec2-client-vpn/pull/115/files#r1853814788

```suggestion
variable "logging_permissions_boundary" {
```

---

done